src/Type/Definition/NonNull.php

```diff
@@ -26,16 +26,20 @@ public function toString(): string
         return $this->getWrappedType()->toString() . '!';
     }
 
-    public function getOfType()
+    /**
+     * @return NullableType&Type
+     */
+    public function getOfType(): Type
     {
-        return Schema::resolveType($this->ofType);
+        /** @var NullableType&Type $type */
+        $type = Schema::resolveType($this->ofType);
+
+        return $type;
     }
 
-    /**
-     * @return (NullableType&Type)
-     */
     public function getWrappedType(bool $recurse = false): Type
     {
+        /** @var NullableType&Type $type */
```

Comment:
is this needed? It's typed above

![image](https://user-images.githubusercontent.com/327717/114263024-65d4c180-99e3-11eb-89a8-b313bae76f3e.png)

Reply:
I would rather omit it, PHPStan will not let me.

Reply:
Aight, let's skip this for now.

Eg. psalm can detect useless `@var` annotations so we can automatically clean this up later